### ISSUE TITLE
Build yq with 1.19.2 to remove various CVEs

### ISSUE THREAD

**Issue by reece-oliver** [mikefarah/yq#1394]
**Describe the bug**

Build with go 1.19.2 to remove various CVEs, currently have various CVEs present which is stopping us ship our product
```
CVE               Severity   PackageName                        PackagePath                     Version              Status

CVE-2022-2880     high       go                                 /usr/local/bin/yq               1.19.1               fixed in 1.19.2, 1.18.7
CVE-2022-2879     high       go                                 /usr/local/bin/yq               1.19.1               fixed in 1.19.2, 1.18.7
CVE-2022-41715    high       go                                 /usr/local/bin/yq               1.19.1               fixed in 1.19.2, 1.18.7
```

Version of yq: 4.28.2

**Comment by reece-oliver** [comment]
Is there any plan to make a new release son? @mikefarah

**Comment by mikefarah** [comment]
yep - fixed in 4.29.1 :)
